common/ruletypes.h

```diff
@@ -315,6 +315,9 @@ RULE_INT(World, MaximumQuestErrors, 30, "Changes the maximum number of quest err
 RULE_INT(World, BootHour, 0, "Sets the in-game hour world will set when it first boots. 0-24 are valid options, where 0 disables this rule")
 RULE_BOOL(World, UseItemLinksForKeyRing, false, "Uses item links for Key Ring Listing instead of item name")
 RULE_BOOL(World, UseOldShadowKnightClassExport, true, "Disable to have Shadowknight show as Shadow Knight (live-like)")
+RULE_BOOL(World, AllowWorldWideSpellsOnBots, false, "Set to true to allow world wide spells (cast/remove) to affect bots")
+RULE_BOOL(World, AllowWorldWideSpellsOnMercs, false, "Set to true to allow world wide spells (cast/remove) to affect mercenaries")
+RULE_BOOL(World, AllowWorldWideSpellsOnPets, false, "Set to true to allow world wide spells (cast/remove) to affect pets")
```

Comment:
Why do we need two sets of almost identical rules that almost do the same thing ?

Reply:
I can consolidate it to three rules.